modules/olm-updating-operator-catalog-image.adoc

```diff
@@ -3,10 +3,10 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 
 ifdef::openshift-origin[]
-:registry-image: quay.io/openshift/origin-operator-registry:4.7.0
+:registry-image: quay.io/openshift/origin-operator-registry:4.8.0
```

Comment:
Same here, https://github.com/openshift/openshift-docs/pull/33720 will remove this module for 4.8, so the changes in this PR can be dropped.